test/y2storage/dialogs/guided_setup/select_filesystem/ng_test.rb

```diff
@@ -68,55 +68,37 @@
   end
 
   describe "skip?" do
-    before do
-      allow(Y2Storage::Dialogs::GuidedSetup).to receive(:allowed?).and_return(allowed)
-    end
-
-    context "when the Guided Setup is not allowed" do
-      let(:allowed) { false }
-
-      let(:volumes) { [double("VolumeSpecification", configurable?: true)] }
+    context "when the proposal settings contain no volumes" do
+      let(:volumes) { [] }
 
       it "returns true" do
-        expect(dialog.skip?).to eq(true)
+        expect(dialog.skip?).to eq true
       end
     end
 
-    context "when the Guided Setup is allowed" do
-      let(:allowed) { true }
-
-      context "and the proposal settings contain no volumes" do
-        let(:volumes) { [] }
+    context "and none of the volumes are configurable" do
```

Comment:
This should start with something line "when" or "if" instead of "and", isn't it? Right now is read as "SelectFilesystem::Ng#skip? and none of the volumes are configurable returns true" instead of "SelectFilesystem::Ng#skip? when none of the volumes are configurable returns true".